dds/DCPS/transport/framework/DataLink.cpp

```diff
@@ -1181,8 +1187,8 @@ operator<<(std::ostream& str, const DataLink& value)
        localId != value.assoc_by_local_.end();
        ++localId) {
     for (RepoIdSet::const_iterator
-         remoteId = localId->second.begin();
-         remoteId != localId->second.end();
+         remoteId = localId->second.associated_.begin();
```

Comment:
This isn't new to your commit, but this statement being split across 2 lines made it hard for me to understand at a glance